sdk/python/tests/dsl/utils_tests.py

```diff
@@ -0,0 +1,66 @@
+from kfp.dsl import ContainerOp
+from kfp.utils import use_secret
+import unittest
+import inspect
+
+
+class TestAddSecrets(unittest.TestCase):
+    def test_default_use_secret(self):
+        spec = inspect.getfullargspec(use_secret)
+        assert len(spec.defaults) == 5
```

Comment:
for asserting errors, shall we use self.assertEqual() etc.? for it offers more informative diff comparison between the golden standard and actual output.